libs/components/indicators/src/lib/modules/key-info/key-info.component.html

```diff
@@ -8,9 +11,14 @@
     <ng-content select="sky-key-info-value" />
   </div>
   <div class="sky-key-info-label sky-field-label sky-font-data-label">
-    <span skyTrim><ng-content select="sky-key-info-label" /></span
+    <span skyId #label="skyId"><ng-content select="sky-key-info-label" /></span
```

Comment:
Why can't we keep `skyTrim` on this level?

Reply:
It doesn't work here because there is an additional wrapper element in between and the trim directive needs to be around the text node.

Reply:
Got it.